TrackletAlgorithm/TrackletProcessor.h

```diff
@@ -398,7 +326,7 @@ ::rinv * const rinv, TrackletParameters::PHI0PAR * const phi0, TC::Types::z0 * c
 
     valid_proj[i] = valid_zmin && valid_zmax && valid_phimax && valid_phimin;
 
-    if (rproj[i] < irprojmaxcut) {
+    if (projectionLayers[Seed][i]<3) {
```

Comment:
@aryd Please use the `layerDisk` enum from Constants.h here:
```suggestion
    if (projectionLayers[Seed][i]<TF::L4) {
```

Reply:
Fixed